server/issue.go

```diff
@@ -169,18 +169,44 @@ func httpAPICreateIssue(ji Instance, w http.ResponseWriter, r *http.Request) (in
 		return http.StatusInternalServerError, errors.Errorf("Failed to create issue. %s", err.Error())
 	}
 
+	// Reply with an ephemeral post with the Jira issue formatted as slack attachment.
 	startLink := fmt.Sprintf("/plugins/%s%s", manifest.Id, routeUserStart)
 	msg := fmt.Sprintf("Created Jira issue [%s](%s/browse/%s) by [mattermost-jira-plugin](%s)", created.Key, ji.GetURL(), created.Key, startLink)
-
-	// Reply to the post with the issue link that was created
+  
```

Comment:
Looks like there is an extra space on this line that is causing CI to fail. @iamsayantan Do you mind removing the space?
```suggestion

```

Reply:
Done.